src/app/common/model/confirmation-modal.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface IConfirmationModalData {
```

Comment:
This one is still in wrong directory -> should be in `/common/confirmation-modal/`.